srcpkgs/anope/template

```diff
@@ -0,0 +1,63 @@
+# Template file for 'anope'
+pkgname=anope
+version=2.0.6
+revision=1
+wrksrc="${pkgname}-${version}-source"
+build_style=cmake
+make_dirs="
+	/etc/anope 755 root root
+	/var/lib/anope 755 _anope _anope
+	/var/log/anope 755 _anope _anope"
+short_desc="Set of IRC Services designed for flexibility and ease of use"
+maintainer="ametisf <ametisf@gmail.com>"
+license="GPL-2.0-only"
+homepage="https://www.anope.org/"
+distfiles="https://github.com/${pkgname}/${pkgname}/releases/download/${version}/${pkgname}-${version}-source.tar.gz"
+checksum=546377755af33f5497a1f5811dfab7d289eaf07b644e4babd1de8d95b9936816
+system_accounts="_anope"
+
+configure_args=" \
```

Comment:
No need for the backslash since it is inside a string, please put it directly below build_style=

no need to define type and  CMAKE_BUILD_TYPE is set automatically